core/tests/snapshots.rs

```diff
@@ -21,6 +21,7 @@ macro_rules! DEFINE_SNAPSHOT_VERSION_PARAMETERIZED_TEST_FUNCTIONS {
             }
 
             #[test]
+            #[ignore]
```

Comment:
Maybe just remove entirely?

Reply:
I got this test working, so ignore is gone.